docs/source/index.rst

```diff
@@ -1,91 +1,3 @@
-.. CFA Multisignal Renewal documentation master file, created by
-   sphinx-quickstart on Sun Mar 17 15:31:47 2024.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+Do not edit this file. The index.rst document is automatically generated via GitHub Actions using the root level `README.md` file.
```

Comment:
Is this successfully commented?

Reply:
It doesn't look like it.

Reply:
I could be more clear. That file (including content) is completely replaced by the README.md -> index.rst file during the website.yaml workflow. I'll make it more explicit.

Reply:
Does that placeholder file definitely need to exist / be tracked in git at all in that case?

Reply:
Yes, because the project's website has a button for "Edit this page on GitHub" which will take the user to that page.